notebooks/05_operationalize/lightgbm_criteo_o16n.ipynb

```diff
@@ -0,0 +1,785 @@
+       "    white-space: pre-wrap;\n",
+       "    word-wrap: break-word;\n",
+       "    word-break: break-all;\n",
+       "    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n",
+       "    font-size: 13px;\n",
+       "    color: #555;\n",
+       "    margin-left: 4px;\n",
+       "    line-height: 19px;\n",
+       "  }\n",
+       "</style>\n",
+       "<div class=\"ansiout\"></div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "MODEL_NAME = 'lightgbm_criteo.mml'  # this name must exactly match the name used to save the pipeline model in the estimation notebook\n",
+    "MODEL_DESCRIPTION = 'LightGBM Criteo Model'\n",
+    "\n",
+    "# Setup AzureML assets (names must be lower case alphanumeric without spaces and between 3 and 32 characters)\n",
+    "# Azure ML Webservice\n",
+    "SERVICE_NAME = 'lightgbm-criteo'\n",
+    "# Azure ML Container Image\n",
+    "CONTAINER_NAME = 'lightgbm-criteo'\n",
+    "CONTAINER_RUN_TIME = 'spark-PY'\n",
+    "# Azure AKS Service\n",
+    "AKS_NAME = 'predict-aks'\n",
+    "\n",
+    "# Names of other files that are used below\n",
+    "CONDA_FILE = \"deploy_conda.yaml\"\n",
+    "DRIVER_FILE = \"mmlspark_serving.py\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Setup AzureML Workspace\n",
+    "Workspace configuration can be retrieved from the portal and uploaded to Databricks<br>\n",
+    "See [AzureML on Databricks](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-configure-environment#azure-databricks)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<style scoped>\n",
+       "  .ansiout {\n",
+       "    display: block;\n",
+       "    unicode-bidi: embed;\n",
+       "    white-space: pre-wrap;\n",
+       "    word-wrap: break-word;\n",
+       "    word-break: break-all;\n",
+       "    font-family: \"Source Code Pro\", \"Menlo\", monospace;;\n",
+       "    font-size: 13px;\n",
+       "    color: #555;\n",
+       "    margin-left: 4px;\n",
+       "    line-height: 19px;\n",
+       "  }\n",
+       "</style>\n",
+       "<div class=\"ansiout\">Found the config file in: /dbfs/aml_config/config.json\n",
+       "Performing interactive authentication. Please follow the instructions on the terminal.\n",
+       "To sign in, use a web browser to open the page https://microsoft.com/devicelogin and enter the code AYP4LT73Y to authenticate.\n",
+       "Interactive authentication successfully completed.\n",
+       "</div>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ws = Workspace.from_config('/dbfs/aml_config/config.json')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prepare the Serialized Model\n",
```

Comment:
Section number starts from 2. 1 is missing.

Reply:
good catch, just going to remove numbering